Sources/InfoView.swift

```diff
@@ -64,7 +64,11 @@ class InfoView: UIVisualEffectView {
     super.init(effect: blurEffect)
 
     [label, imageView, borderView].forEach {
-      addSubview($0)
```

Comment:
When I used this solution I have a crash because you adding this before if condition 
2018-05-07 11:39:40.849131+0300 Joker[7443:4905523] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: '<UILabel: 0x105b2c8f0; frame = (0 0; 0 0); userInteractionEnabled = NO; layer = <_UILabelLayer: 0x1c428b810>> has been added as a subview to <BarcodeScanner.InfoView: 0x105a3d9e0; baseClass = UIVisualEffectView; frame = (0 0; 0 0); layer = <CALayer: 0x1c02238c0>>. Do not add subviews directly to the visual effect view itself, instead add them to the -contentView.'

Solution:- just comment this line and this solution will work as a charm.

Reply:
The solution:- 
init() {
    let blurEffect = UIBlurEffect(style: .extraLight)
    super.init(effect: blurEffect)
    
    [label, imageView, borderView].forEach {
             // addSubview($0)
                if #available(iOS 11.0, *) {
                       contentView.addSubview($0)
                   } else {
                       addSubview($0)
                    }
    }
    
    status = Status(state: .scanning)
  }